lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -138,3 +147,4 @@ def _get_s6a_manager(service, processor):
 
 if __name__ == "__main__":
     main()
+    
```

Comment:
this blank line should not contain any whitespaces

Reply:
Comment addressed.